lib/datasource/git-refs/index.ts

```diff
@@ -32,23 +32,34 @@ export async function getRawRefs({
     }
 
     // fetch remote tags
-    const lsRemote = await git.listRemote([lookupName, '--sort=-v:refname']);
+    const lsRemote = await git.listRemote([
+      '--sort=-v:refname',
+      '--tags',
+      '--heads',
```

Comment:
```suggestion
      '--refs',
```

This will remove all pseudo refs
https://git-scm.com/docs/git-ls-remote.html

Reply:
I initially tried that, but this will also add other things like merge requests which are not useful actually
```
16c8ebeab61d5f86a6ea1f2bfec80a65d1cfead9	refs/heads/master
ac23d58cf398f0c31a05dea5e5d78dea6663cba4	refs/heads/renovate/configure
2b7667fb5f40b32ae7d52c88c5cc295f52b97898	refs/pipelines/1/head
2b7667fb5f40b32ae7d52c88c5cc295f52b97898	refs/environments/1/head
2b7667fb5f40b32ae7d52c88c5cc295f52b97898	refs/pipelines/1/head
f3ec126e31331019ec135c4c093f6f249c04a7c6	refs/merge-requests/1/merge
84c41c73ebf1fcce52a3de07682d07473943f92a	refs/merge-requests/2/head
c689af53089e7409f69e0469d9bdd422ab8f8da8	refs/merge-requests/2/merge
e8b5c41ddf75770ed29d5f924a686052bb376bab	refs/tags/1.2.0
ee85705ee6d770a66cc143dc1a3d49de410965ba	refs/tags/1.3.0
16c8ebeab61d5f86a6ea1f2bfec80a65d1cfead9	refs/tags/2.0.0
```

Reply:
See updated suggestion

Reply:
adding --refs to --heads and --tags will not help us here, since only tags and heads are required for now. adding --refs for the sake seems not necessary. I think it would be better to add more flags only if they are required.

Reply:
`--refs` will remove rows like `bd237d0c9c55f42e5ef8a2ca1d5e99f30662097a        refs/tags/v8.9.3^{}`

Reply:
til about annotated tags, thanks!